react/src/pages/SessionLauncherPage.tsx

```diff
@@ -1295,67 +1304,190 @@ const SessionLauncherPage = () => {
                           {currentProject.name}
                         </Descriptions.Item>
                         <Descriptions.Item label={t('general.Image')}>
-                          <Row
-                            style={{ flexFlow: 'nowrap', gap: token.sizeXS }}
-                          >
-                            <Col>
-                              <ImageMetaIcon
-                                image={
-                                  form.getFieldValue('environments')?.version ||
-                                  form.getFieldValue('environments')?.manual
-                                }
-                              />
-                            </Col>
-                            <Col>
-                              {/* {form.getFieldValue('environments').image} */}
-                              <Flex direction="row">
-                                {form.getFieldValue('environments')?.manual ? (
-                                  <Typography.Text
-                                    code
-                                    style={{ wordBreak: 'break-all' }}
-                                    copyable={{
-                                      text: form.getFieldValue('environments')
-                                        ?.manual,
-                                    }}
-                                  >
-                                    {form.getFieldValue('environments')?.manual}
-                                  </Typography.Text>
-                                ) : (
-                                  <>
-                                    <SessionKernelTags
-                                      image={
+                          {supportExtendedImageInfo ? (
+                            <Row
+                              style={{ flexFlow: 'nowrap', gap: token.sizeXS }}
```

Comment:
It's super minor thing, but looks a little far between image and string
```suggestion
                              style={{ flexFlow: 'nowrap' }}
```